src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -214,6 +246,11 @@ public DatasetTerm toDatasetTerm(Dataset dataset) {
       User user = userDAO.findUserById(dataset.getCreateUserId());
       term.setCreateUserId(dataset.getCreateUserId());
       term.setCreateUserDisplayName(user.getDisplayName());
+      term.setSubmitter(toUserTerm(user));
+    });
+    Optional.ofNullable(dataset.getUpdateUserId()).ifPresent(userId -> {
+      User user = userDAO.findUserById(dataset.getUpdateUserId());
+      term.setUpdateUser(toUserTerm(user));
     });
```

Comment:
Another `Optional` feature is that it supports a few streaming-like APIs, like `map()`, `filter()`, `flatMap()` etc. So another way to write this is:
```suggestion
    Optional.ofNullable(dataset.getUpdateUserId())
        .map(userDAO::findUserById)
        .map(this::toUserTerm)
        .ifPresent(term::setUpdateUser);
```